src/components/Atoms/Input/index.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useRef, useEffect } from "react";
+import styled from "styled-components";
+import COLOR from "../../../variables/color";
+import TEXT from "../../../variables/texts";
+import FONTFAMILY from "../../../variables/font_family";
+
+const Input = ({ onEditComplete, defaultValue }) => {
+  const ref = useRef(null);
+
+  useEffect(() => {
+    let value = null;
+    if (defaultValue == null) {
```

Comment:
```suggestion
    if (defaultValue === undefined) {
```
基本的に厳密等価を使うのが望ましいと思います。
また、propsに何も渡さない場合 undefined になるので、書き直すとしたら上のような感じになるかと思います。